default/buildings.txt

```diff
@@ -625,7 +625,7 @@ BuildingType
             stackinggroup = "INDUSTRY_CENTER_STACK"
             effects = SetTargetIndustry value = Value + Target.Population * 3 * [[INDUSTRY_PER_POP]]
     ]
-    icon = ""
+    icon = "icons/tech/industrial_centre_ii.png"
```

Comment:
I want to mention about workflow:
This changes was done in separate commit and it is good. But this commit is part of this pull request and I don't like it. I suggest to make separate pr for this changes.